### ISSUE TITLE
teminal-window resize => dashboard unusable

### ISSUE THREAD

**Issue by pixelass** [FormidableLabs/webpack-dashboard#213]
When resizing the terminal window the entire layout of the dashboard is broken
Scollbars don't work old paints are not removed. Basicallly fully unusable once resized.

![webpack-dashboard](https://user-images.githubusercontent.com/1148334/31891223-0fc871dc-b806-11e7-83af-448761f78608.gif)

![webpack-dashboard_2](https://user-images.githubusercontent.com/1148334/31891287-3d192636-b806-11e7-992f-0e6b3880352b.gif)

###### Steps to reproduce the problem

* start dashboard
* resize terminal
* => scrollbars etc are misplaced => dashboard unusable
###### Please provide a gist of relevant files

1. package.json (specifically the script you are using to start the dashboard)
2. webpack.config.js
3. index.js (Your express based dev server, if applicable)

====================================================================

###### More Details

- OS X 10.12.6 (16G29)
- OS X terminal
- "webpack-dashboard": "^1.0.0-7"
- xterm-256color

**Comment by ryan-roemer** [comment]
Please upgrade to `^1.0.0` and report back if that doesn't fix the problem. (I have full resize support + scrollbars w/ 1.0 on a nearly identical setup to what you have).

**Comment by pixelass** [comment]
I updated the package but the issue persists.

**Comment by jbinto** [comment]
I also noticed this when upgrading from `0.3.0` to `1.0.0`.  Ended up removing `webpack-dashboard` just to get moving along. I suspected something in `blessed` changed, but looks like it's the same version.

Will try to send a reproduction case in a few hours.

**Comment by cvsguimaraes** [comment]
Can confirm the exact same behavior with `gnome-terminal`.
Command: `webpack-dashboard -- webpack-dev-server --public spm:8080 --hot`
Versions:
 - `webpack@^3.4.x`
 - `webpack-dev-server@^2.7.1`
 - `webpack-dashboard@^1.0.0-5`

**Comment by ryan-roemer** [comment]
@cvsguimaraes -- You're still on the beta. `^1.0.0` is the current. 

If anyone can get us a minimal repository with install + reproduction steps I'll take a look. Thanks!

**Comment by jbinto** [comment]
Tried to get a reproduction, but I am now seeing it on `0.3.0` on a vanilla `create-react-app`. 

I suspected upgrading to `1.0.0` caused this, but that doesn't seem to be case. Now I suspect a transitive dependency, or something weird about iTerm2 or OS X terminal settings that has changed in the last ~3 weeks.

Will continue to investigate.

**Comment by pixelass** [comment]
@jbinto I'm still on Sierra.  (See first post: OS X 10.12.6 (16G29))

I tried node 6.x and 8.x

**Comment by ryan-roemer** [comment]
Hmmm.... I've tried the repository (thanks @jbinto !) in both terminal and iterm2 on my mac os 10.12.6 (16G1036) and everything resizes correctly for me with workable scrollbars the whole way through.

@carlospaelinck -- Any ideas on how we can reproduce what these other folks are seeing on our machines?

**Comment by ryan-roemer** [comment]
Just to be sure can everyone experiencing the issue(s) output the installed blessed version?

```sh
$ cat node_modules/blessed/package.json | grep "\"version\""
  "version": "0.1.81",
```

is mine.

**Comment by rkistinger** [comment]
Experiencing the same issues as @jbinto with near identical setup.

@ryan-roemer 
`cat node_modules/blessed/package.json | grep "\"version\""
  "version": "0.1.81"`

**Comment by pixelass** [comment]
@ryan-roemer `"version": "0.1.81"`

Very weird. I just tested the test repo from @jbinto and could not reproduce the issue. I then went back to the project where I first saw the issue. Without updating anything or reinstalling the node_modules suddenly it worked there too. Not quite sure where to go from there. 
The only difference was the network I was connected to. 
(I have not made any system updates or similar since the issue was posted)
